tests/modeladmin/tests.py

```diff
@@ -615,6 +617,49 @@ def test_log_actions(self):
         self.assertEqual(ma.log_change(mock_request, self.band, 'changed'), LogEntry.objects.latest('id'))
         self.assertEqual(ma.log_change(mock_request, self.band, 'deleted'), LogEntry.objects.latest('id'))
 
+    @mock.patch('django.contrib.admin.options.get_deleted_objects')
+    def test_get_deleted_objects_called(self, mock_delobjs):
+        # the get_deleted_objects() function imported to
+        # django.contrib.admin.options from django.contrib.admin.utils is
+        # called by modeladmin's get_deleted_objects() method (#17962).
```

Comment:
Ticket references are reserved for obscure issues (see [Coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)). Should this inline comment be a docstring?